src/managers/NewsManager.js

```diff
@@ -0,0 +1,81 @@
+const ReqUtil = require('../util/ReqUtil');
+const CacheManager = require('./CacheManager');
+const News = require('../structures/News');
+
+class NewsManager{
+    constructor(client){
+        /**
+         * The client instance.
+         * @type {Client}
+         */
+        this.client = client;
+
+        /**
+         * The cache manager
+         * @type {CacheManager} 
+         * @private
+         */
+        this._cache = new CacheManager(client, News);
+    }
+
+    /**
+     * Fetches a Trackmania splashscreen and returns its data.
+     * @param {News} newsId The splashscreen ID
+     * @param {Boolean} cache Whether to get the news from cache or not
+     * @returns {Promise<News>} The splashscreen
+     * @example 
+     * client.news.get(143).then(news => {
+     *     console.log(news.title);
+     * });
+     */
+    async get(newsId, cache = this.client.options.cache.enabled){
```

Comment:
Are you sure this is the only use case? You'd have to know the ID before you even attempt to call `get()`, right? Shouldn't there also be a kind of singleton `getAll()` so you can just get all available news?

Reply:
Yeah, i'll add that later (after we done all stucts)

Reply:
Fair enough, just checking 👍